Pdu.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include <cstdint>
+#include<string>
+#include <bitset>
+
+class Pdu
+{
+private:
+	static constexpr int SIZE_BYTES = 5;
+	static constexpr int NAME_SIZE = 30;
+	static constexpr int MESSAGE_SIZE = 1024;
+	uint32_t _size;
+	char _name[NAME_SIZE];
+	char _message[MESSAGE_SIZE];
+	char* _buffer;
```

Comment:
`char *` is a c pointer - here we do c++
i'd rather you use `std::string`, `std::array`, `std::vector` or a smart pointer instead, depending on the use-case